generator/internal/descriptor_utils.cc

```diff
@@ -652,15 +652,37 @@ std::string GetEffectiveServiceName(VarsDictionary const& vars,
   return std::string(name.data(), name.size());
 }
 
+/// If a service name mapping exists, return the replacement comment.
+/// If it does not exist, return a null optional.
+/// Parses a command line argument in the form:
+/// {"service_name_to_comments": "service_a=comment_a,service_b=comment_b"}.
+absl::optional<std::string> GetReplacementComment(VarsDictionary const& vars,
```

Comment:
If `GetReplacementComment()` and `GetEffectiveServiceName()` don't need to be declared in `descriptor_utils.h`, then they should be defined within the anonymous namespace, and nor do they deserve doxygen-level comments.

Reply:
Moved to anon namespace and fixed the comments.

Didn't realize the `///` was a doxygen thing.